src/annotator/components/AdderToolbar.js

```diff
@@ -34,55 +55,60 @@ function NumberIcon({ badgeCount }) {
  */
 function AdderToolbarArrow({ arrowDirection }) {
   return (
-    <Icon
-      name="pointer"
-      classes={classnames(
-        // Position the arrow in the horizontal center at the bottom of the
-        // container (toolbar). Note: the arrow is pointing up at this point.
-        'absolute left-1/2 -translate-x-1/2',
-        // Override `1em` width/height rules in `Icon` to size the arrow as
-        // its SVG dimensions dictate
-        'h-auto w-auto z-2',
-        'text-grey-3 fill-white',
+    <div
```

Comment:
Here's an example of me adding an additional parent component to handle the positioning, and not applying that styling directly to the icon component. This is less of a real concern with icons, which have no classes applied to them other than what you set on `className`, but it's more hygienic IMHO.